test/functionalTest/cases/1697_more_than_one_BadInput/more_than_one_BadInput.test

```diff
@@ -0,0 +1,135 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Yeah, I usually don't mix underscore style and camel case, but I made an exception here as *BadInput* is a 'concept'.   Probably a bad idea, changing it.